Gutenberg/cocoapods_helpers.rb

```diff
@@ -53,20 +53,24 @@ def gutenberg_pod
 end
 
 def gutenberg_local_pod
-  options = { path: local_gutenberg_path }
-
-  raise "Could not find Gutenberg pod at #{options[:path]}. You can configure the path using the #{LOCAL_GUTENBERG_KEY} environment variable." unless File.exist?(options[:path])
-
-  puts "[Gutenberg] Installing pods using local Gutenberg version from #{local_gutenberg_path}"
+  options_gb = gutenberg_pod_options(name: 'Gutenberg', path: local_gutenberg_path)
+  options_aztec = gutenberg_pod_options(name: 'RNTAztecView', path: "#{local_gutenberg_path}/gutenberg/packages/react-native-aztec/")
```

Comment:
Nitpick: We can omit the last slash in the path.
```suggestion
  options_aztec = gutenberg_pod_options(name: 'RNTAztecView', path: "#{local_gutenberg_path}/gutenberg/packages/react-native-aztec")
```